### PR TITLE
Chub 2696

### PR DESCRIPTION
**Pull request by bnouvel-wio**

### CODE REVIEW

File: resolution/controllers/hub_key_handler.py [hunk 1]
```diff
@@ -228,18 +228,18 @@ def get(self):
...
-                self.redirect(redirect)
-            elif 'application/json' in self.request.headers.get('Accept', '').split(';'):
-                self.write(parsed_key)
-            else:
-                self.render('template.html', hub_key=parsed_key)
-        except Exception, e:
-            self.set_status(500)
-            self.write(str(type(e))+":"+repr(e))
+        except ValueError:
+            self.set_status(404)
```
Comment (tconway) [inline resolution/controllers/hub_key_handler.py:232]: We should raise HTTP Error so that it gets turned into the right json format.

Comment (tconway) [inline resolution/controllers/hub_key_handler.py:232]: Ignore us, this is fine

**Comment by tconway** [comment]
Bamboo Job?

**Comment by bnouvel-wio** [comment]
http://bamboo.digicat.io/browse/CHCL-MPT181-1

**Pull request merged**